### ISSUE TITLE
En chiffres: Add historique anciennes législatures

### ISSUE THREAD

**Issue by awenig** [datanfr/datan#543]

**Comment by IGLOU-EU** [comment]
Effectivement, se baser uniquement sur les votes de la 16eme induit un biais dans l'étude des comportements des partis, surtout que cette législature fut très courte.

**Comment by awenig** [comment]
Oui nous avons déjà ouvert des pages historiques par groupe politique (https://datan.fr/groupes/legislature-16/lfi-nupes/statistiques).

L'idée est de le faire pour les pages "députés", ainsi que pour la section "En chiffres" du site (https://datan.fr/statistiques).

Mais c'est du taff...
